src/utils/mesh.js

```diff
@@ -58,12 +61,12 @@ function getCSaaSMeshConfig (core, githubOrg, githubRepo) {
                 }
                 },
                 "responseConfig": {
-                    "headers": ["X-Magento-Cache-Id"]
+                "headers": ["X-Magento-Cache-Id"]
                 }
             }
             ]
         }
-    }
+        }
```

Comment:
Same here, this indented when I wouldn't expect. Is the config valid? Are we missing some closing bracket?